atest/robot/libdoc/cli.robot

```diff
@@ -11,10 +11,16 @@ Default format is got from output file extension
     String ${OUTBASE}.libspec    XML:HTML     String    path=${OUTBASE}.libspec
 
 Using --format overrides output file extension
-    -f XmL ${TESTDATADIR}/resource.robot ${OUTHTML}    XML         resource
-    --format hTmL BuiltIn ${OUTBASE}.xxx               HTML        BuiltIn     path=${OUTBASE}.xxx
-    --format XML:HTML String ${OUTXML}                 XML:HTML    String      path=${OUTXML}
-    --format XML String ${OUTBASE}.libspec             XML         String      path=${OUTBASE}.libspec
+    -f XmL ${TESTDATADIR}/resource.robot ${OUTHTML}       XML         resource
+    --format hTmL BuiltIn ${OUTBASE}.xxx                  HTML        BuiltIn     path=${OUTBASE}.xxx
+    --format XML String ${OUTBASE}.libspec                XML         String      path=${OUTBASE}.libspec
+
+Using --specdocformat to specify doc format in output
+    --format XML --specdocformat RAW String ${OUTXML}              XML         String      path=${OUTXML}
+    --format XML --specdocformat HTML String ${OUTXML}             XML:HTML    String      path=${OUTXML}
+    --format XML --specdocformat RAW String ${OUTBASE}.libspec     XML         String      path=${OUTBASE}.libspec
+    --format XML --specdocformat HTML String ${OUTBASE}.libspec    XML:HTML    String      path=${OUTBASE}.libspec
```

Comment:
Using `XML:HTML` here as an expected output is a bit weird when the format is otherwise removed. Would be good to change the test to pass these values separately but if that's a big task this is ok as well.